src/test/java/org/eclipse/uprotocol/communication/NotifierExample.java

```diff
@@ -0,0 +1,52 @@
+package org.eclipse.uprotocol.communication;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import org.junit.jupiter.api.Test;
+
+import org.eclipse.uprotocol.transport.UListener;
+import org.eclipse.uprotocol.transport.UTransport;
+import org.eclipse.uprotocol.v1.UCode;
+import org.eclipse.uprotocol.v1.UMessage;
+import org.eclipse.uprotocol.v1.UUri;
+
+public class NotifierExample {
+    
+    // Notification topic
+    private UUri topic = UUri.newBuilder()
+        .setUeId(4).setUeVersionMajor(1).setResourceId(0x8000).build();
+
+    @Test
+    public void testSendNotification() {
+        UTransport transport = new TestUTransport();
+        
+        
+        // Who to send the notification to
+        final UUri destination = UUri.newBuilder()
+            .setUeId(3).setUeVersionMajor(1).build();
+
+        // Send the notification (without payload)
+        assertEquals(Notifier.notify(transport, topic, destination), UCode.OK);
+    }
+
+    @Test
+    public void testReceivingNotifications() {
+        // Test transport
+        final TestUTransport transport = new TestUTransport();
+        final UListener listener = new UListener() {
+            @Override
+            public void onReceive(UMessage message) {
+                // Handle receiving notifications here
+                assertNotNull(message);
+            }
+        };
+
+        // Register listener to receive notifications
+        assertEquals(Notifier.registerNotificationListener(transport, topic, listener),
+            UCode.OK);
+
+        // Unregister the listener so we no longer receive notifications
+        assertEquals(Notifier.unregisterNotificationListener(transport, topic, listener),
```

Comment:
Where do you send notification to receive in this test?

Reply:
Example code only shows how to use the API